server/service/src/sync/test/test_data/purchase_order_line.rs

```diff
@@ -128,17 +131,20 @@ fn purchase_order_line_push_record() -> TestSyncOutgoingRecord {
             line_number: 1,
             item_link_id: "item_a".to_string(),
             item_name: "Item A".to_string(),
-            quan_rec_to_date: 5000.0,
-            delivery_date_requested: Some(NaiveDate::from_ymd_opt(2018, 3, 19).unwrap()),
-            delivery_date_expected: None,
-            snapshot_quantity: 0.0,
-            packsize_ordered: 1000.0,
-            quan_original_order: 100.0,
-            quan_adjusted_order: None,
+            received_number_of_units: 5000.0,
```

Comment:
Sorry I nit used serde rename cause easier for me to match the fields with what it is in OG

Reply:
Personally I prefer the other pattern as it is then clearer what fields map to what in the translations but I agree it's easier to read in the tests cases.

Reply:
Oh not with tests, I mean with what they are in the OG UI since everything had been renamed~